topdown/time.go

```diff
@@ -93,7 +106,7 @@ func builtinDate(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) er
 }
 
 func builtinClock(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
-	t, err := tzTime(operands[0].Value)
+	t, _, err := tzTime(operands[0].Value)
```

Comment:
I was briefly wondering if this affected the existing builtins -- it doesn't, but I've found that when the argument array is not known at compile-time, any length of it will be accepted:
```
% echo '[1670411354472815000, "GMT", "hi", "team"]' | opa-0.46.1 eval -I -fpretty 'time.date(input)' --strict-builtin-errors
[
  2022,
  12,
  7
]
```

This is **unrelated** to this PR, just something I've stumbled upon when reviewing this.